doc/source/user_guide/case.rst

```diff
@@ -82,16 +82,58 @@ The valid sections for setting up a nekRS simulation are:
 
 Each of the keys and value types are described below. 
 
-**TODO** Description of keys in table
+nekRS supports both traditional CPU-based architectures and accelerated 
+computing on GPUs. In the ``OCCA`` section, users can select various backend options, 
+such as the ``serial`` mode for CPU-only computations, ``CUDA`` for accelerated 
+computing with Nvidia GPUs, or ``HIP`` for AMD GPUs.
 
 .. code-block:: cpp
 
   std::string user_occa_backend;
   options.getArgs("THREAD MODEL", user_occa_backend);
 
-In other words, if you have ``backend = CUDA`` in the ``.par`` file, then
+As shown in snippet above, if you have ``backend = CUDA`` in the ``.par`` file, then
 ``user_occa_backend`` would be set to ``CUDA`` in the above code.
 
+In the ``GENERAL`` section, key settings include configuring the simulation's polynomial 
+order (``polynomialOrder``), restart options, and time iteration parameters. For nekRS 
+simulations, a recommended polynomial order is ``p=7``, though the software currently 
```

Comment:
Is the notation consistent? Maybe need to work with theory?
Personally, I like $N$ and will use $p$ as number of processor.